io.github.maurycyliebner.enve.yaml

```diff
@@ -0,0 +1,186 @@
+app-id: io.github.maurycyliebner.enve
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: enve
+rename-desktop-file: enve.desktop
+rename-icon: enve
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+cleanup-commands:
+  - rm -rf /app/{include,sbin,share/{man,cmake},lib/{*.a,*.la,python2.7,mkspecs,pkgconfig,qt5/mkspecs,plugins/designer}}
```

Comment:
Any reason why this and not regular `cleanup` array?

Reply:
https://github.com/flatpak/flatpak-builder/issues/233

(I wanted to combine all in one.)